vllm/worker/multi_step_model_runner.py

```diff
@@ -215,6 +217,46 @@ def prepare_model_input(
         )
         return model_input
 
+    def _async_process_outputs(self, model_input: StatefulModelInput,
```

Comment:
Very cool combination of maybe_pythonize and output_proc_call_back. These two features worked out well together!

Reply:
Yeah, thanks for building the maybe_pythonize structure :)